lib/scrolls/log.rb

```diff
@@ -238,7 +238,6 @@ def mtx
     def sync_stream(out=nil)
       out = STDOUT if out.nil?
       s = out
-      s.sync = true
```

Comment:
The method name here indicates that it's `sync`. Could we change the method name or write a second method that's async?